src/phoebus_driver.cpp

```diff
@@ -351,6 +352,28 @@ TaskCollection PhoebusDriver::RungeKuttaStage(const int stage) {
     }
   }
 
+  // Extra per-step user work
+  TaskRegion &sync_region_5 = tc.AddRegion(num_partitions);
```

Comment:
You can have as many partitions or as few as you want on a per-region basis. Just set this to 1. Then you have to create a new meshdata object specifically for this task region that just owns all blocks.

Reply:
Good workaround, but yeah I should just do the right thing